res/controllers/Pioneer-DDJ-200-scripts.js

```diff
@@ -46,14 +131,219 @@ DDJ200.seek = function(channel, control, value, status, group) {
+    engine.setValue(vgroup, "super1", val);
+};
+
+DDJ200.cueDefault = function(channel, control, value, status, group) {
+    if (value) { // only if button pressed, not releases, i.e. value === 0
+        var vDeckNo = DDJ200.vDeckNo[script.deckFromGroup(group)];
+        var vgroup = "[Channel" + vDeckNo +"]";
+        if (DDJ200.vDeck[vDeckNo]["jogDisabled"]) {
+            engine.setValue(vgroup, "cue_set", true);
+            print("true");
+        } else {
+            engine.setValue(vgroup, "cue_gotoandplay", true);
+        }
+        var deckNo = script.deckFromGroup(group);
+        midi.sendShortMsg(0x90 + deckNo - 1, 0x0C, 0x7F *    // set cue LED
+                          (engine.getValue(vgroup, "cue_point") !== -1));
+        // set play LED
+        midi.sendShortMsg(status, 0x0B, 0x7F * engine.getValue(vgroup, "play"));
+    }
+};
+
+DDJ200.cueGotoandstop = function(channel, control, value, status, group) {
+    if (value) { // only if button pressed, not releases, i.e. value === 0
+        var vDeckNo = DDJ200.vDeckNo[script.deckFromGroup(group)];
+        var vgroup = "[Channel" + vDeckNo +"]";
+        engine.setValue(vgroup, "cue_gotoandstop", true);
+        //engine.setValue(vgroup, "start_stop", true); // go to start if prefered
+        midi.sendShortMsg(status, 0x0B, 0x7F * engine.getValue(vgroup, "play"));
+    }
+};
+
+DDJ200.hotcueNActivate = function(channel, control, value, status, group) {
+    var vDeckNo = DDJ200.vDeckNo[script.deckFromGroup(group)];
+    var vgroup = "[Channel" + vDeckNo +"]";
+    var hotcue = "hotcue_" + (control + 1);
+    engine.setValue(vgroup, hotcue + "_activate", true);
+    midi.sendShortMsg(status, control,
+        0x7F * engine.getValue(vgroup, hotcue + "_enabled"));
+    var deckNo = script.deckFromGroup(group);
+    midi.sendShortMsg(0x90 + deckNo - 1, 0x0B, 0x7F *
+                      engine.getValue(vgroup, "play")); // set play LED
+};
+
+DDJ200.hotcueNClear = function(channel, control, value, status, group) {
+    var vDeckNo = DDJ200.vDeckNo[script.deckFromGroup(group)];
+    var vgroup = "[Channel" + vDeckNo +"]";
+    engine.setValue(vgroup, "hotcue_" + (control + 1) + "_clear", true);
+    midi.sendShortMsg(status-1, control, 0x00);        // set hotcue LEDs
+};
+
+DDJ200.pfl = function(channel, control, value, status, group) {
+    if (value) { // only if button pressed, not releases, i.e. value === 0
+        var deckNo = script.deckFromGroup(group);
+        var vDeckNo = DDJ200.vDeckNo[deckNo];
+        var vgroup = "[Channel" + vDeckNo +"]";
+        var pfl = ! engine.getValue(vgroup, "pfl");
+        engine.setValue(vgroup, "pfl", pfl);
+        if (DDJ200.fourDeckMode === false) {
+            midi.sendShortMsg(status, 0x54, 0x7F * pfl);  // switch pfl LED
+        }
+    }
+};
+
+DDJ200.switchLEDs = function(vDeckNo) {
+    // set LEDs of controller deck according to virtual deck
+    var c = 1; if (vDeckNo % 2) c = 0;
```

Comment:
missing braces and multiple statements on the same line. Also, please use more meaningful variable names, e.g.: `channelOffset` or something like that.